src/services/commit/useCommitTeam.tsx

```diff
@@ -0,0 +1,395 @@
+})
+
+const RepositorySchema = z.object({
+  __typename: z.literal('Repository'),
+  commit: CommitSchema.nullable(),
+})
+
+const RequestSchema = z.object({
+  owner: z
+    .object({
+      repository: z.discriminatedUnion('__typename', [
+        RepositorySchema,
+        RepoNotFoundErrorSchema,
+        RepoOwnerNotActivatedErrorSchema,
+      ]),
+    })
+    .nullable(),
+})
+
+const query = `query GetCommitTeam(
+  $owner: String!
+  $repo: String!
+  $commitid: String!
+  $filters: ImpactedFilesFilters
+) {
+  owner(username: $owner) {
+    repository(name: $repo) {
+      __typename
+      ... on Repository {
+        commit(id: $commitid) {
+          state
+          commitid
+          pullId
+          branchName
+          createdAt
+          author {
+            username
+          }
+          uploads {
+            edges {
+              node {
+                id
+                state
+                provider
+                createdAt
+                updatedAt
+                flags
+                jobCode
+                downloadUrl
+                ciUrl
+                uploadType
+                buildCode
+                name
+                errors {
+                  edges {
+                    node {
+                      errorCode
+                    }
+                  }
+                }
+              }
+            }
+          }
+          message
+          ciPassed
+          compareWithParent {
+            __typename
+            ... on Comparison {
+              indirectChangedFilesCount
+              directChangedFilesCount
+              state
+              patchTotals {
+                coverage: percentCovered
+              }
+              impactedFiles(filters: $filters) {
+                headName
+                missesCount
+                patchCoverage {
+                  coverage: percentCovered
+                }
+              }
+            }
+            ... on FirstPullRequest {
+              message
+            }
+            ... on MissingBaseCommit {
+              message
+            }
+            ... on MissingHeadCommit {
+              message
+            }
+            ... on MissingComparison {
+              message
+            }
+            ... on MissingBaseReport {
+              message
+            }
+            ... on MissingHeadReport {
+              message
+            }
+          }
+        }
+      }
+      ... on NotFoundError {
+        message
+      }
+      ... on OwnerNotActivatedError {
+        message
+      }
+    }
+  }
+}`
+
+interface UseCommitTeamArgs {
+  provider: string
+  owner: string
+  repo: string
+  commitid: string
+  filters?: {
+    hasUnintendedChanges?: boolean
+    flags?: Array<string>
+    ordering?: z.infer<typeof ImpactedFilesOrdering>
+  }
+  refetchInterval?: number
+}
+
+export function useCommitTeam({
+  provider,
+  owner,
+  repo,
+  commitid,
+  filters = {},
+  refetchInterval = 2000,
```

Comment:
I believe we don't need this refetch interval/polling related information for this hook. This was one of the earliest hooks we made and it was kind of an "all you can shove" kind of hook, so it would be ideal to get rid of this functionality if it's not an immense lift, up to you as this was probably mostly copy paste from the existing code 👌

Reply:
I don't see how we can not have it, if a commit has not finished processing then we went to poll until it has completed, so the user does not have to do anything (i.e. refreshing the page) to get the new information.

Reply:
Gotcha gotcha, I was confusing this with the useResyncUser and thought this was logic inside of it. Sgsgsg